src/current/molt/molt-fetch.md

```diff
@@ -361,11 +362,11 @@ By default, MOLT Fetch moves all data from the [`--source`](#source-and-target-d
 --schema-filter 'public'
 ~~~
 
-`--table-filter` specifies a range of tables to move to CockroachDB, formatted as a POSIX regex string. For example, to move every table in the source database that has "user" in the title:
+`--table-filter` and `--table-exclusion-filter` specify tables to include and exclude from the migration, respectively, formatted as POSIX regex strings. For example, to move every source table that has "user" in the title and exclude every source table that has "temp" in the title:
```

Comment:
nit: from my experience, tables have "names" rather than "titles". For example, their are "table naming conventions".

```suggestion
`--table-filter` and `--table-exclusion-filter` specify tables to include and exclude from the migration, respectively, formatted as POSIX regex strings. For example, to move every source table that has "user" in the table name and exclude every source table that has "temp" in the table name:
```